static/js/sections.js

```diff
@@ -242,6 +242,18 @@ function downloadOperationReport() {
 
 /** ADVERSARIES **/
 
+function toggleAdversaryView() {
+    $('#viewAdversary, #new_phase').toggle();
+    $('#addAdversary').toggle();
+
+
+    if ($('#togBtnOp').is(':checked')) {
+        showHide('.queueOption,#opBtn', '#adversaries');
```

Comment:
we'll want to replace the ".queueOption,#opBtn" from each of these calls. The way to read this would be, "show the first parameter, hide the second".

Reply:
Ah - cool that makes sense.